README.md

```diff
@@ -1,8 +1,19 @@
-# UCL ARC Portal
+# UCL ARC Services Portal
 
 Management portal for [UCL ARC](https://www.ucl.ac.uk/advanced-research-computing/)
 services.
 
+Production site: https://portal.arc.ucl.ac.uk/
+
+Staging site: https://portal.ucl-arc.dev/
+
+# Overview
+
+The Service Portal aims to serve as a single entry point for researchers and other users to manage and engage with the various services and platforms provided by ARC, eg. TRE, DSH.
```

Comment:
- should be careful with nomenclature here. platform vs. service vs. environment
- DSH isn't provided by ARC
- TLAs deserve links